modules/ROOT/pages/prereq-aws-terraform.adoc

```diff
@@ -64,18 +67,21 @@ ssh -i 'anypoint.pem' ec2-user@18.221.155.87
 | AWS_REGION | Specifies the AWS region where Terraform creates the cluster, for example, `us-east-2`.
 | TF_VAR_ssh_user=<SSH_USER> | Specifies the ssh user, for example, `ec2-user`, `centos`, etc.
 | TELEKUBE_CLUSTER_NAME | Specifies the name of the cluster and the corresponding AWS resources.
-| TELEKUBE_NODE_PROFILE_COUNT_node | Specifies the number of nodes in your cluster. Possible values are `3` and `6`.
-| TELEKUBE_NODE_PROFILE_INSTANCE_TYPE_node | Must be `m5.2xlarge`.
 | TELEKUBE_NODE_PROFILES | Must be set to `node`.
+| TELEKUBE_NODE_PROFILE_COUNT_node | Specifies the number of nodes in your cluster. Possible values are `4` and `7`.
+| TELEKUBE_NODE_PROFILE_INSTANCE_TYPE_node | Must be `m5.2xlarge`.
+| TELEKUBE_NODE_PROFILE_COUNT_node_amv | Specifies the number of nodes for the AMV addon in your cluster. Must be `3` if planning to install the AMV addon. Omit otherwise.
+| TELEKUBE_NODE_PROFILE_INSTANCE_TYPE_node_amv | Must be `m5.4xlarge` if planning to install the AMV addon. Omit otherwise.
```

Comment:
Must be `m5.8xlarge`

Reply:
Changed to: `Must be `m5.8xlarge` if planning to install the AMV addon. Omit otherwise.`